Samples/BatchSupport/BatchSupport/BatchSupport.vcxproj

```diff
@@ -74,10 +74,12 @@
     <ClCompile>
       <Optimization>Disabled</Optimization>
       <PreprocessorDefinitions>_DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <AdditionalIncludeDirectories Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">..\..\SampleSharedLib\SampleSharedLib;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
Looks like you only are adding the shared lib for Debug|x64. This should be applied to all platform|configuration combinations